recipe/meta.yaml

```
@@ -1,23 +1,28 @@
 {% set name = "black" %}
-{% set version = "24.10.0" %}
+{% set version = "25.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 846ea64c97afe3bc677b761787993be4991810ecc7a4a937816dd6bddedc4875
+  sha256: 33496d5cd1222ad73391352b4ae8da15253c5de89b93a80b3e2c8d9a19ec2666
+  patches:
+    - patches/0001-Fix-f-string-format-spec-CI-failure-4690.patch
 
 build:
   number: 0
-  skip: True # [py<39]
   script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
   entry_points:
     - black = black:patched_main
     - blackd = blackd:patched_main
+  skip: True # [py<39]
 
 requirements:
+  build:
+    - patch  # [unix]
+    - m2-patch  # [win]
```

```suggestion
    - msys2-patch  # [win]
```
`msys2-` is more modern than `m2-`